aces/Loris
# [issue_tracker] Fix incorrect url to issues in notification email

## Brief summary of changes

Currently, the notification email sent by the issue tracker module has the old url . This PR fixes this issue by refactoring the code from an ajax directory to issue_tracker/Edit.

#### Testing instructions (if applicable)

1. Login to loris as admin account. Go to the Issue Tracker module and edit an issue. Select a user that has your email associated with the account in the 'Add others to watching?' field and click on 'Update Issue'. You will receive a 'Change to Issue' email notification. If you don't receive one, edit another field and 'Update Issue' again. Once you receive the email, click on the url in that email and notice that it ends with something like issue_tracker/issue/?issueID=78. This is the old way of accessing issues which will now just take you to the 'Add Issue' page. The url should be issue_tracker/issue/78.
2. On this PR branch, repeat the above step. The email you receive should have issue_tracker/issue/78 instead. Click on the link, and make sure it takes you to the correct issue.

#### Link(s) to related issue(s)

* Resolves #  (Reference the issue this fixes, if any.)
